hazelcast/src/test/java/com/hazelcast/cache/nearcache/NearCacheRecordStoreTest.java

```diff
@@ -196,13 +196,13 @@ public void evictionTriggeredAndHandledSuccessfullyWithEntryCountMaxSizePolicyAn
     }
 
     @Test
-    public void evictionTriggeredAndHandledSuccessfullyWithEntryCountMaxSizePolicyAndDefaultEvictionPolicyOnNearCacheObjectRecordStore() {
-        doEvictionWithEntryCountMaxSizePolicy(InMemoryFormat.OBJECT, null);
+    public void evictionTriggeredAndHandledSuccessfullyWithEntryCountMaxSizePolicyAndRandomEvictionPolicyNearCacheObjectRecordStore() {
```

Comment:
As I said before, just fixing bugs here. TLC for Near Cache tests is another huge task which we'll plan for this sprint, if there is time left.